segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -220,6 +219,7 @@ protected void doStart() {
         if (this.durableLog.isOffline()) {
             // Attach a listener to the DurableLog's awaitOnline() Future and initiate the services' startup when that
             // completes successfully.
+            log.info("{}: DurableLog is OFFLINE. Not starting secondary services yet.");
```

Comment:
This has an extra `{}`, or a missing argument in the call.

Reply:
Fixed. Thank you.